### PR TITLE
Fix CS issues and add tests for ACO Trees with prefixes

### PR DESCRIPTION
**Pull request by dakota**
This carries on from #49 by adding more tests, ensuring existing tests pass and fixing code standard issues.

### CODE REVIEW

**Comment by markstory** [comment]
Looks good :+1:

File: src/AclExtras.php [hunk 1]
```diff
@@ -325,22 +449,14 @@ protected function _checkMethods($className, $controllerName, $node, $pluginPath
             if (strpos($action, '_', 0) === 0) {
                 continue;
             }
-            $path = $this->rootNode . '/' . $pluginPath . $controllerName . '/' . $prefix . $action;
-            $this->_checkNode($path, $prefix . $action, $node->id);
-            $methods[$key] = $prefix . $action;
+            $path = str_replace(':controller', $controllerName, $template);
+            $path = str_replace(':action', $action, $path);
```
Comment (rchavik) [inline src/AclExtras.php]: How about replacing these two lines with:
 
```php
$path = str_replace([':controller', ':action'], [$controllerName, $action], $template);
```

Comment (dakota) [inline src/AclExtras.php]: Done :)

**Comment by dakota** [comment]
Ok to merge?

**Comment by darxmac** [comment]
looks like this would fix #74 ?

**Comment by rchavik** [comment]
Superseded by #71

**Pull request closed without merging**